### ISSUE TITLE
[playground] When lint is skipped, no result is returned.

### ISSUE THREAD

**Issue by kagankan** [markuplint/markuplint#1332]
## Describe the bug

Markuplint skips the lint execution if the filename does not match the parser settings.

```
{
  "parser": {
    "\\.hoge$": "@markuplint/svelte-parser"
  }
}
```

The playground should report that lint was skipped.
